spec/actors/curation_concerns/apply_order_actor_spec.rb

```diff
@@ -50,17 +83,18 @@
     end
 
     context 'with in_works_ids' do
-      let(:attributes) { { ordered_member_ids: [] } }
+      let(:attributes) { { ordered_member_ids: ["Blah4"] } }
       let(:root_actor) { double }
       before do
         allow(CurationConcerns::Actors::RootActor).to receive(:new).and_return(root_actor)
         allow(root_actor).to receive(:update).with({}).and_return(true)
-        g = child
-        g.save
         curation_concern.apply_depositor_metadata(user.user_key)
         curation_concern.save!
       end
       it "attaches the parent" do
+        child.title = ["Generic Title"]
```

Comment:
Why did this stuff move out of the `before` block.  It seems like that's the natural place for it.